contrib/epee/src/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ file(GLOB EPEE_HEADERS_PUBLIC
 	"${EPEE_INCLUDE_DIR_BASE}/**/*.h*" # Any number of subdirs will be included.
 )
 
-add_library(epee STATIC byte_slice.cpp byte_stream.cpp hex.cpp abstract_http_client.cpp http_auth.cpp mlog.cpp net_helper.cpp net_utils_base.cpp string_tools.cpp
+monero_add_library(epee byte_slice.cpp byte_stream.cpp hex.cpp abstract_http_client.cpp http_auth.cpp mlog.cpp net_helper.cpp net_utils_base.cpp string_tools.cpp
```

Comment:
The major change was the removal of the `static` keyword ... ? With that change, a cmake configuration request for dynamic libraries will build `epee` as a dynamic library regardless. So the moving of the macro is more for consistency?

Reply:
Moving the macro to a broader scope allowed to use it here, in the Epee.

The macro doesn't expect the linkage type, as this is being configured by the user. It may as well be static, if the user chooses so. The common macro allows for other things as well.